jarm-main/src/main/java/database/user/UserDAO.java

```diff
@@ -0,0 +1,95 @@
+package database.user;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+import database.DBController;
+import database.DBStatements;
+import de.jarm.main.data.User;
+import de.jarm.main.database.exceptions.UserExistsException;
+import de.jarm.main.database.exceptions.WrongUserOrPasswordException;
+import utils.DateUtils;
+
+public class UserDAO {
+	
+	public List<User> loadUsers() throws Exception {
+		List<User> users = new ArrayList<User>();		
+
+		try (Connection con = DBController.getInstance().getConnection();
+				Statement stmt = con.createStatement();){
+			
+			ResultSet rs = stmt.executeQuery(DBStatements.SELECT_ALL_USERS);
+			
+			while (rs.next()) {
+				User u = new User(rs.getInt("id"), rs.getString("name"), rs.getString("password"), rs.getString("email"));
+				users.add(u);
+			}
+			
+			return users;
+		} catch (Exception e) {
+			throw e;
+		}
+	}
+	
+	public User logIn(String email, String password) throws Exception {
+		try (Connection con = DBController.getInstance().getConnection();
+				PreparedStatement pstmt = con.prepareStatement(DBStatements.SELECT_USER_LOGIN)){
+			
+			pstmt.setString(1, email);
+			pstmt.setString(2, password);
+			
+			ResultSet rs = pstmt.executeQuery();
+			User u = new User(rs.getInt("id"), rs.getString("name"), rs.getString("password"), rs.getString("email"));

```

Comment:
if (!rs.next()) {
    throw new WrongUserOrPasswordException();
}
Lasse ihn nicht vorsätzlich in Exceptions laufen.

Reply:
ist eingebaut